packages/common/tests/chains.spec.ts

```diff
@@ -159,7 +159,7 @@ tape('[Common/Chains]: Initialization / Chain params', function (t: tape.Test) {
       const c = new Common({ chain: network })
       const dnsNetworks = c.dnsNetworks()
       st.ok(Array.isArray(dnsNetworks), 'is an array')
-      st.equal(typeof dnsNetworks[0], 'string', 'returns the DNS ENR url as a string')
+      st.equal(typeof dnsNetworks?.[0], 'string', 'returns the DNS ENR url as a string')
```

Comment:
Ah, this answers a bit my question on the "any -> something" renamings. If this makes changes like this necessary we should likely not do it in this round.

Reply:
Fixed by adding `!` to 
```
  dnsNetworks(): string[] {
    return this._chainParams['dnsNetworks']!
  }
``` 
And therefore removing the `undefined` return type